docs/src/basic_usage.md

```diff
@@ -0,0 +1,27 @@
+# Basic usage
+
+You can calculate Riemannian gradient of a function defined in its embedding in multiple ways.
+For example, `rb_onb_fd51` corresponds to a finite differencing scheme and `rb_onb_fwdd` calculates gradient using `ForwardDiff.jl`. [`DifferentiationInterface.jl`](https://github.com/JuliaDiff/DifferentiationInterface.jl) is used to select the backend.
+
+```@example
+using ManifoldDiff
+using DifferentiationInterface
+using Manifolds, FiniteDifferences, ForwardDiff, Zygote
+
+rb_onb_fd51 = ManifoldDiff.TangentDiffBackend(AutoFiniteDifferences(central_fdm(5, 1)))
+rb_onb_fwdd = ManifoldDiff.TangentDiffBackend(AutoForwardDiff())
+rb_proj_zyg = ManifoldDiff.RiemannianProjectionBackend(AutoZygote())
+
+s2 = Sphere(2)
+
+A = [1.0 2.0 5.0; 2.0 -1.0 4.0; 5.0 4.0 0.0]
+
+f(p) = p' * A * p
+q = [0.0, 1.0, 0.0]
+
+println(ManifoldDiff.gradient(s2, f, q, rb_onb_fd51))
+println(ManifoldDiff.gradient(s2, f, q, rb_onb_fwdd))
+println(ManifoldDiff.gradient(s2, f, q, rb_proj_zyg))
+```
```

Comment:
I think this would be a bit better as a quarto, since then one would see the result here as well; but we can maybe also set that up in a next PR

Reply:
I think we can set up quarto when we have a bit more to put there. For such a simple example I think Documenter.jl is fine.